### ISSUE TITLE
Plotting reference levels

### ISSUE THREAD

**Issue by sspina** [strengejacke/sjPlot#402]
Is there a way to plot reference level of factors as well using plot_model()?  
-Thank you

**Comment by strengejacke** [comment]
No, currently not. I wonder what would be the benefit of including the ref. level in a plot? It would be a dot at position 0 (or 1 for Odds Ratios, e.g.)? I think this makes plots rather unclear - or is it common in certain disciplines?

**Comment by sspina** [comment]
No, it's just to make it easier to read the plots for an unexperienced audience.
Thank you for your answer!

**Comment by strengejacke** [comment]
I must think about an easy way how to optionally plot reference levels, so I leave this issue open.

**Comment by xieguagua** [comment]
option forreference levels will be great , thanks.

**Comment by roxannebeauclair** [comment]
Just to add to this, is it possible to add reference levels to the tab_model(). That would be very useful, so that people can see what the reference level is for ORs.

**Comment by jakubdrapal** [comment]
Just adding to previously expressed opinions that to plot a reference level would make interpretation of plots for laypublic easier.
